### DISCUSSION TITLE
Problem about the output along track (track_i.ww3)

### DISCUSSION THREAD

**Discussion by ffayalac** [NOAA-EMC/WW3#731] [Q&A] [answered]
Hi everyone,

We are currently modelling a test case in the Caribbean, and we are trying
to get the spectra along a hurricane track. According to what we
understand, firstly, the track points and their corresponding dates must be
specified in the track_i.ww3 file; secondly, the dates and the flag (T)
must be activated in the ww3_shel.inp and finally, ww3_trnc.inp (or
ww3_trck.inp) must be edited.

We follow these steps and get a .nc file with the information described in
the user guide; however, the longitude and latitudes in this file don't
match with the location data in track_i.ww3. In fact, the lats and lons in
the .nc file seem to be in accordance with the specified resolution in
ww3_grid, and it seems as if we had to interpolate them.

What should it be the correct process to obtain spectra along a track? Can
we get spectra for locations in the track that do not match with the
spatial resolution (e.g. spatial resolution of 0.25 and a point in the
track with lon and lat of 273.653 and 15.247, respectively)?

**Comment by aliabdolali** [comment, marked as answer]
@FrankAyalaC my understanding is that track is only available on nodes of the computational grid coordinates, close to prescirbed coordinates of the track. So, the only option I see is that you interpolate from outputs on grids nodes to track coordinates (i.e. you can define grid nodes on both sides of the track in ww3_shel and then interpolate from the generated outputs to the actual track).
